pennylane/interfaces/tfe.py

```diff
@@ -12,184 +12,25 @@
-
-We can convert the default NumPy-interfacing QNode to a TensorFlow-interfacing QNode by
-using the :meth:`~.QNode.to_tfe` method:
-
->>> qnode1 = qnode1.to_tfe()
->>> qnode1
-<QNode: device='default.qubit', func=circuit, wires=2, interface=TensorFlow>
-
-Internally, the :meth:`~.QNode.to_tfe` method uses the :func:`~.TFEQNode` function
-to do the conversion.
-
-
-Quantum gradients using TensorFlow
-----------------------------------
-
-Since a TensorFlow-interfacing QNode acts like any other TensorFlow function,
-the standard method used to calculate gradients in eager mode with TensorFlow can be used.
-
-For example:
-
-.. code-block:: python
-
-    dev = qml.device('default.qubit', wires=2)
-
-    @qml.qnode(dev, interface='tfe')
-    def circuit(phi, theta):
-        qml.RX(phi[0], wires=0)
-        qml.RY(phi[1], wires=1)
-        qml.CNOT(wires=[0, 1])
-        qml.PhaseShift(theta, wires=0)
-        return qml.expval(qml.PauliZ(0))
-
-    phi = tfe.Variable([0.5, 0.1])
-    theta = tfe.Variable(0.2)
-
-    grad_fn = tfe.implicit_value_and_gradients(circuit)
-    result, [(phi_grad, phi_var), (theta_grad, theta_var)] = grad_fn(phi, theta)
-
-Now, printing the gradients, we get:
-
->>> phi_grad
-array([-0.47942549,  0.        ])
->>> theta_grad
--5.5511151231257827e-17
-
-.. _pytf_optimize:
-
-Optimization using TensorFlow
------------------------------
-
-To optimize your hybrid classical-quantum model using the TensorFlow eager interface,
-you **must** make use of the TensorFlow optimizers provided in the ``tf.train`` module,
-or your own custom TensorFlow optimizer. **The** :ref:`PennyLane optimizers <optimization_methods>`
-**cannot be used with the TensorFlow interface, only the** :ref:`numpy_qnode`.
-
-For example, to optimize a TFE-interfacing QNode (below) such that the weights ``x``
-result in an expectation value of 0.5, we can do the following:
-
-.. code-block:: python
-
-    import tensorflow as tf
-    import tensorflow.contrib.eager as tfe
-    tf.enable_eager_execution()
-
-    import pennylane as qml
-
-    dev = qml.device('default.qubit', wires=2)
-
-    @qml.qnode(dev, interface='tfe')
-    def circuit(phi, theta):
-        qml.RX(phi[0], wires=0)
-        qml.RY(phi[1], wires=1)
-        qml.CNOT(wires=[0, 1])
-        qml.PhaseShift(theta, wires=0)
-        return qml.expval(qml.PauliZ(0))
-
-    phi = tfe.Variable([0.5, 0.1], dtype=tf.float64)
-    theta = tfe.Variable(0.2, dtype=tf.float64)
-
-    opt = tf.train.GradientDescentOptimizer(learning_rate=0.1)
-    steps = 200
-
-    for i in range(steps):
-        with tf.GradientTape() as tape:
-            loss = tf.abs(circuit(phi, theta) - 0.5)**2
-            grads = tape.gradient(loss, [phi, theta])
-
-        opt.apply_gradients(zip(grads, [phi, theta]), global_step=tf.train.get_or_create_global_step())
-
+This module contains the :func:`TFEQNode` function to convert Numpy-interfacing quantum nodes with TensorFlow
+Eager mode compatible quantum nodes.
```

Comment:
This module contains the :func:`TFEQNode` function that converts Numpy-interfacing quantum nodes to TensorFlow
Eager mode compatible quantum nodes.